### DISCUSSION TITLE
infuse添加webdav报错

### DISCUSSION THREAD

**Discussion by bingbaicai** [AlistGo/alist#2110] [General]
Alist V3上面已经成功挂载了阿里和百度
但是在Infuse7 添加webdav地址后报错

**Comment by mh567** [comment]
+1，遇到同样的错误。

**Reply by Dashan-IZ** [reply]
你们成功了嘛？

**Comment by bingbaicai** [comment]
成功了&nbsp;


275456082
***@***.***



&nbsp;




------------------&nbsp;原始邮件&nbsp;------------------
发件人: ***@***.***&gt;; 
发送时间: 2022年11月18日(星期五) 凌晨2:33
收件人: ***@***.***&gt;; 
抄送: ***@***.***&gt;; ***@***.***&gt;; 
主题: Re: [alist-org/alist] infuse添加webdav报错 (Discussion #2110)





 
你们成功了嘛？
 
&mdash;
